packages/cdk-routing-split/src/delete/handler.ts

```diff
@@ -1,25 +1,41 @@
 import { APIGatewayEvent, Context } from 'aws-lambda';
 import { createLogger, Logger, serializeError } from '@daysmart/aws-lambda-logger';
-import { HttpError } from '../http-error';
 import { action } from './action';
+import { HttpError } from '../http-error';
+import { DeleteRequest, Key } from './interface';
 
-export const handler = async (event: APIGatewayEvent, context: Context): Promise<any> => {
+export const deleteRecord = async (event: APIGatewayEvent, context: Context): Promise<any> => {
```

Comment:
So far, we called this just delete. You can rename this one to delete and update references to it accordingly.